blueman/plugins/applet/Menu.py

```diff
@@ -4,18 +4,94 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-from blueman.Functions import *
+import dbus
```

Comment:
```import dbus.service``` still not signal after this though, functions are being called properly just no signal emission 😞